example/ios/reactnativemembranewebrtcexample/reactnativemembranewebrtcexample.entitlements

```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-  <dict>
-    <key>aps-environment</key>
-    <string>development</string>
-    <key>com.apple.security.application-groups</key>
-    <array>
-      <string>group.com.membrane.reactnativemembrane</string>
-    </array>
-  </dict>
-</plist>
\ No newline at end of file
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
+	<key>com.apple.security.application-groups</key>
+	<array/>
```

Comment:
you removed group from here and it's probably going to break screencast in the example